spec/ruby/core/kernel/format_spec.rb

```diff
@@ -11,4 +11,24 @@
   it "is accessible as a module function" do
     Kernel.format("%s", "hello").should == "hello"
   end
+
+  it "formats string with precision" do
+    Kernel.format("%.3s", "hello").should == "hel"
+    Kernel.format("%-3.3s", "hello").should == "hel"
+  end
+
+  describe "on multibyte strings" do
+    it "formats string with precision" do
+      skip "pending https://github.com/oracle/truffleruby/issues/2314"
+
+      Kernel.format("%.3s", "hello".encode('UTF-16LE')).should == "hel"
+    end
+
+    it "preserves encoding" do
+      skip "pending https://github.com/oracle/truffleruby/issues/2314"
```

Comment:
@chrisseaton is this the right way to tag something as "skip until the issue is fixed?"

Reply:
No, sorry. Can you remove the `skip` lines, and then run `jt tag spec/ruby/core/kernel/format_spec.rb`. The tags need to live in separate files as the specs are shared between all Ruby implementations. You can then add a comment in the tag file.